src/control/cmd/daos/health.go

```diff
@@ -14,12 +14,16 @@ import (
 
 	"github.com/daos-stack/daos/src/control/build"
 	"github.com/daos-stack/daos/src/control/cmd/daos/pretty"
+	"github.com/daos-stack/daos/src/control/common/cmdutil"
 	"github.com/daos-stack/daos/src/control/lib/daos"
+	"github.com/daos-stack/daos/src/control/lib/ranklist"
+	"github.com/daos-stack/daos/src/control/lib/ui"
 	"github.com/daos-stack/daos/src/control/logging"
 )
 
 type healthCmds struct {
-	Check healthCheckCmd `command:"check" description:"Perform DAOS system health checks"`
+	Check   healthCheckCmd `command:"check" description:"Perform DAOS system health checks"`
+	NetTest netTestCmd     `command:"net-test" description:"Perform non-destructive DAOS networking tests"`
```

Comment:
I wanted to convey to the user that the test won't overwrite any data, even though it's doing writes. Simpler to just write "non-destructive" to allay any concerns, otherwise the concerned user has to go read documentation to figure out whether or not they should be worried.